enhancements/etcd/automated-backups-no-config.md

```diff
@@ -0,0 +1,299 @@
+Using a CR enables users to override the default backup configuration.
+
+By design, the **No-Config** backup approach works _orthogonal_ to the [Cron](https://github.com/openshift/enhancements/blob/master/enhancements/etcd/automated-backups) based method, and not a replacement.
+However, this method is designed to work on all variants of Openshift (e.g. Single-node and Bare-metal), as well as having backups on all control-plane nodes, for recovering from disaster scenarios.
+
+### API Extensions
+
+This proposal uses the same [API](https://github.com/openshift/enhancements/blob/master/enhancements/etcd/automated-backups.md#api-extensions) from the Cron based method.
+No additional API changes are required.
+
+### Topology Considerations
+TBD
+#### Hypershift / Hosted Control Planes
+This proposal does not affect HyperShift.
+#### Standalone Clusters
+TBD
+#### Single-node Deployments or MicroShift
+TBD
+### Workflow Description
+TBD
+
+## Design Details
+
+To back up the Etcd datastore and cluster resources of an Openshift cluster for the purpose of restoration, a persistent storage must be utilised.
+Several main point have been considered during designing the implementation of this proposal, however the storage solution to be used is the most influential, particularly to be used by all Openshift variants.  
+
+### Storage
+
+Ideally, the storage solution to use should have the following characteristics :-
+
+- Backups should be taken on each master node to avoid overwhelming a specific etcd member more than others.
+- Spreading the backups among all etcd cluster members provide guarantees for disaster recovery in case of losing two members at the same time.
+- One fits all storage solution, to be used across all Openshift variants including Bare metal and Single Node Openshift.
+
+According to these requirement, only `hostPath` and `localVolume` storage could be utilized. 
+However, if the master node where the `PV` is mounted became unhealthy/unavailable/unreachable, then the backups are no longer accessible, also taking new backups is no longer possible.
+Hence, `hostPath` has been used as it is the only viable option that could be utilized on all Openshift variants. 
+
+### Workload
+
+In order to take backups for Etcd data store and cluster resources, a workload must run on the cluster while having access to Etcd and cluster resources.
+Since Etcd data store of an Openshift cluster runs on `Pod` similar to other workload, several constructs such as `Deployment`, `DaemonSet`, `StatefulSet` and `SideCar` could be used.
+
+This section discusses two approaches (i.e. `SideCar` and `DaemonSet`), which are implemented and tested, explaining the Pros and Cons of each method.  
+Before deciding upon these approaches, several approaches have been investigated. See [Alternatives](#Alternatives).
```

Comment:
> Are there concerns about having this disk space available? Do we need to make sure there is overhead on control plane nodes? 32GB is not trivially available by default I would wager

Added to open questions. However, the pruning functionality has an option to prune the backups according to size

Reply:
> Before merging, one of these should be moved to the alternatives section

+1, so @hasbro17 wanted to open the discussion about which approach to use before deciding the final one, that's why I have added both of them here with their Pros&Cons 

I personally vote for the `DaemonSet` approach, since deploying the side car within etcd static pod involves rolling out new revision upon enabling / disabling, and also imposing a risk of bringing the whole etcd cluster down upon any error from the backup container